.github/workflows/build_doc.yml

```diff
@@ -155,7 +155,7 @@ jobs:
         if: ${{ failure() && steps.get_round.outputs.result != 'stop' }}
         with:
           script: |
-            const error = "${{steps.build_and_run.outputs.DoxygenError}}"
+            const error = `${{steps.build_and_run.outputs.DoxygenError}}`
```

Comment:
That will not work. In the run that failed, the content of `DoxygenError` was:
```
warning: tag INPUT: input source `/home/runner/work/cgal/cgal/Mesh_3/include//CGAL/Detect_features_in_image.h'' does not exist
warning: tag INPUT: input source `/home/runner/work/cgal/cgal/Mesh_3/include//CGAL/Detect_features_on_image_bbox.h'' does not exist
warning: source /home/runner/work/cgal/cgal/Mesh_3/include//CGAL/Detect_features_in_image.h is not a readable file or directory... skipping.
warning: source /home/runner/work/cgal/cgal/Mesh_3/include//CGAL/Detect_features_on_image_bbox.h is not a readable file or directory... skipping.
```

It contains the characters <code>`</code> and <code>'</code>. Actually, we do not know what will be inside. I think you should follow the advice from [the section **Using an intermediate environment variable** from the Github page **Security hardening for GitHub Actions**](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable). See also https://github.com/actions/github-script#use-env-as-input for using an environment variable with a Github script.